.gitlab-ci.yml

```diff
@@ -226,6 +246,15 @@ examples-test:
         cargo test --verbose --manifest-path ${example}/Cargo.toml;
       done
 
+examples-test-experimental-engine:
+    stage:                           examples
+    <<:                              *docker-env
+    needs:
+        - job:                         clippy-std
+          artifacts:                   false
+    script:
+        cargo test --features ink-experimental-engine --verbose --manifest-path erc20/Cargo.toml;
```

Comment:
Why not
`cargo test --no-default-features --features std, ink-experimental-engine`
?